specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-05-01/examples/FrontdoorList.json

```diff
@@ -0,0 +1,217 @@
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/frontendEndpoint1"
+                      },
+                      {
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/frontendEndpoints/default"
+                      }
+                    ],
+                    "acceptedProtocols": [
+                      "Http"
+                    ],
+                    "patternsToMatch": [
+                      "/*"
+                    ],
+                    "routeConfiguration": {
+                      "@odata.type": "#Microsoft.Azure.FrontDoor.Models.FrontdoorForwardingConfiguration",
+                      "customForwardingPath": "",
+                      "forwardingProtocol": "MatchRequest",
+                      "backendPool": {
+                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/backendPools/backendPool1"
+                      }
+                    },
+                    "rulesEngine": {
+                      "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/rulesEngines/rulesEngine1"
+                    },
+                    "enabledState": "Enabled"
+                  }
+                }
+              ],
+              "healthProbeSettings": [
+                {
+                  "name": "healthProbeSettings1",
+                  "properties": {
+                    "path": "/",
+                    "protocol": "Http",
+                    "intervalInSeconds": 120,
+                    "enabledState": "Enabled",
+                    "healthProbeMethod": "HEAD"
+                  }
+                }
+              ],
+              "loadBalancingSettings": [
+                {
+                  "name": "loadBalancingSettings1",
+                  "properties": {
+                    "sampleSize": 4,
+                    "successfulSamplesRequired": 2
+                  }
+                }
+              ],
+              "backendPools": [
+                {
+                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/backendPools/backendPool1",
+                  "name": "backendPool1",
+                  "properties": {
+                    "backends": [
+                      {
+                        "address": "w3.contoso.com",
+                        "httpPort": 80,
+                        "httpsPort": 443,
+                        "enabledState": "Enabled",
+                        "weight": 1,
+                        "priority": 2
+                      },
+                      {
+                        "privateLinkResourceId": "/subscriptions/subid/resourcegroups/rg1/providers/Microsoft.Network/privateLinkServices/pls1",
+                        "privateLinkLocation": "eastus",
```

Comment:
Changes from API Version `2020-01-01`: Added new fields `privateLinkResourceId` and `privateLinkLocation`